packages/compass-components/src/components/resizeable-sidebar.tsx

```diff
@@ -11,18 +11,28 @@ const containerStyles = css({
   maxWidth: '80%',
   height: '100%',
   position: 'relative',
+});
+
+const containerStylesDark = css({
   background: uiColors.gray.dark3,
   color: 'white',
 });
```

Comment:
If you're feeling fancy here, you can define your palette for the modes in this one place with custom variables:


```suggestion
const containerStylesDark = css({
  '--background-color': uiColors.gray.dark3,
  '--color': 'white',
  '--icon-color': 'white',
  '--icon-color-hover': 'white',
  '--item-color': 'white',
  '--item-color-hover': 'white',
  '--item-color-active': 'white'
  '--item-bg-color': uiColors.gray.dark2,
  '--item-bg-color-hover': uiColors.gray.dark3,
  '--item-bg-color-active': uiColors.gray.dark1,
  backgroundColor: 'var(--background-color)',
  color: 'var(--color)',
});
```

... and then just use these properties inside the components with no need to do the javascript logic for this in multiple places

Reply:
Makes sense. Didn't think of using css vars. Will change. Although I'm not 100% sure if this will work with the icon and svg path thingies?

Reply:
It gets a little bit hairy with some things having darkMode passed in as an attribute. I'll see.

Reply:
Turns out you _can_ pass className to icons. Also weirdly it is using `color`, not `fill`. I guess the fill react prop is used as the `color` property internally somewhere.